rust/linux-client/ansible/everything.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Run from `rust/linux-client`. Builds the client and uses Ansible to set it up on
+# ReactorScram's VM, and install `systemd-resolved`. Doesn't work in CI or anything.
```

Comment:
If this is specific to your VM we shouldn't include it in the repo, you can use (.git/info/exclude)[https://stackoverflow.com/a/1753078] to skip it in the commits.

Reply:
I'll hide all the Ansible stuff then, since it probably won't be officially used